backend/src/main.ts

```diff
@@ -1,20 +1,22 @@
-import { NestFactory } from "@nestjs/core";
-import { AppModule } from "app.module";
-import { ConfigService } from "@nestjs/config";
-import { Logger, ValidationPipe } from "@nestjs/common";
+import {NestFactory} from "@nestjs/core";
+import {AppModule} from "app.module";
+import {ConfigService} from "@nestjs/config";
+import {Logger, ValidationPipe} from "@nestjs/common";
 import * as dotenv from "dotenv";
 import * as dotenvExpand from "dotenv-expand";
 
 (async () => {
-  const logger = new Logger("Main");
-  dotenvExpand.expand(dotenv.config({ path: "./.env" }));
-  const app = await NestFactory.create(AppModule);
-  const configService = app.get(ConfigService);
-  const PORT = +configService.get("SERVER_PORT");
+    const logger = new Logger("Main");
+    dotenvExpand.expand(dotenv.config({path: "./.env"}));
 
-  app.setGlobalPrefix("api");
-  app.useGlobalPipes(new ValidationPipe());
+    const app = await NestFactory.create(AppModule);
+    const configService = app.get(ConfigService);
+    const PORT = +configService.get("SERVER_PORT");
 
-  await app.listen(PORT);
-  logger.log(`Server running on PORT ${PORT}`);
+    app.enableCors();
```

Comment:
It should be dependent on environment variables.

Reply:
it does it

Reply:
How? Because there is no check for this.

Reply:
Look above it, to PORT const